server/sonar-server/src/main/java/org/sonar/server/permission/PermissionTemplateService.java

```diff
@@ -155,6 +156,13 @@ public void applyDefault(DbSession dbSession, ComponentDto component, @Nullable
     indexProjectPermissions(dbSession, asList(component.uuid()));
   }
 
+  public boolean hasProjectCreator(DbSession dbSession, ComponentDto component) {
```

Comment:
what do you think of `hasDefaultTemplateAProjectCreatorPermission` as a method name?